subgraphs/openzeppelin-governor/schema.graphql

```diff
@@ -1,3 +1,39 @@
+type DelegateChange @entity {
```

Comment:
Can we try using `@entity(immutable: true)`? It's supposedly faster to index https://thegraph.com/docs/en/developing/creating-a-subgraph/#defining-entities